spec/bolt/node/ssh_spec.rb

```diff
@@ -98,6 +106,35 @@
         ssh.connect
       end
     end
+
+    it "takes < 2 seconds if connection fails to non-SSH port", ssh: true do
+      bogus_port = 5555
+      Process.spawn("nc -l #{bogus_port}")
```

Comment:
This is gross.. probably want to try to use something cross-platform, like `TCPServer.new`.  Also, can remove `ssh: true`